pkg/webui/components/unit-input/common/duration.js

```diff
@@ -27,10 +27,13 @@ const units = [
   { label: sharedMessages.hours, value: 'h' },
 ]
 
-const encoder = (value, unit) => (value ? `${value}${unit}` : unit)
```

Comment:
Saving just the unit would require transformation logic if the fields should allow empty values. Rather, the field should correctly output null and at the same time keep the last used unit in the state. This way, the unit can be changed even when no integer was entered.